### PR TITLE
support motan

### PR DESCRIPTION
**Pull request by tydhot**
complex motion object will be supported later

### CODE REVIEW

File: soul-admin/src/main/resources/META-INF/schema.sql [hunk 1]
```diff
@@ -281,6 +281,7 @@ INSERT IGNORE INTO `plugin` (`id`, `name`, `role`,`config`, `enabled`, `date_cre
 INSERT IGNORE INTO `plugin` (`id`, `name`, `role`, `enabled`, `date_created`, `date_updated`) VALUES ('14', 'context_path', '1','1', '2020-11-09 01:19:10', '2020-11-09 01:19:10');
 INSERT IGNORE INTO `plugin` (`id`, `name`, `role`,`config`, `enabled`, `date_created`, `date_updated`) VALUES ('15', 'grpc', '1','{"multiSelectorHandle":"1","multiRuleHandle":"0"}','0', '2020-11-09 01:19:10', '2020-11-09 01:19:10');
 INSERT IGNORE INTO `plugin` (`id`, `name`, `role`, `enabled`, `date_created`, `date_updated`) VALUES ('16', 'redirect', '1','0', '2020-11-09 01:19:10', '2020-11-09 01:19:10');
+INSERT IGNORE INTO `plugin` (`id`, `name`, `role`,`config`, `enabled`, `date_created`, `date_updated`) VALUES ('17', 'motan', '1','{"register":"127.0.0.1:2181"}','0', '2020-11-09 01:19:10', '2020-11-09 01:19:10');
```
Comment (dengliming) [inline soul-admin/src/main/resources/META-INF/schema.sql:284]: We also need to add this to `schema.h2.sql`?

File: soul-plugin/soul-plugin-motan/src/main/java/org/dromara/soul/plugin/motan/proxy/MotanProxyService.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+//    private GenericMessage buildGenericMessage(String name, Map<String, Object> map) {
+//        GenericMessage message = new GenericMessage();
+//        message.setName(name);
+//        for (Map.Entry<String, Object> e : map.entrySet()) {
+//            int nameIndex = CommonSerializer.getHash(e.getKey());
+//            message.putFields(nameIndex,e.getValue());
+//        }
+//        return message;
+//    }
```
Comment (dengliming) [inline soul-plugin/soul-plugin-motan/src/main/java/org/dromara/soul/plugin/motan/proxy/MotanProxyService.java:95]: Should we remove this?

Comment (tydhot) [reply soul-plugin/soul-plugin-motan/src/main/java/org/dromara/soul/plugin/motan/proxy/MotanProxyService.java:95]: These codes will be used to support complex object in Motan later. Refer to https://github.com/weibocom/motan/issues/892#issuecomment-599349769

File: soul-admin/src/main/resources/META-INF/schema.sql (hunk 1 above)
Comment (tydhot) [reply soul-admin/src/main/resources/META-INF/schema.sql:284]: ok

File: soul-plugin/soul-plugin-motan/src/main/java/org/dromara/soul/plugin/motan/proxy/MotanProxyService.java (hunk 2 above)
Comment (dengliming) [reply soul-plugin/soul-plugin-motan/src/main/java/org/dromara/soul/plugin/motan/proxy/MotanProxyService.java:95]: OK Let's keep this for now.

**Review by dengliming — APPROVED** [review]

**Pull request merged**